<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D_author</class>
 <widget class="QDialog" name="D_author">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>作者详情</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>合作者：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>著作：</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_essay">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>371</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>booktitle</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tw_partner">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>371</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="b1">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>570</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
